frame/transaction-payment/rpc/runtime-api/src/lib.rs

```diff
@@ -30,5 +31,7 @@ sp_api::decl_runtime_apis! {
 	{
 		fn query_info(uxt: Block::Extrinsic, len: u32) -> RuntimeDispatchInfo<Balance>;
 		fn query_fee_details(uxt: Block::Extrinsic, len: u32) -> FeeDetails<Balance>;
+		fn query_call_info(encoded_call: Bytes, len: u32) -> Result<RuntimeDispatchInfo<Balance>, RuntimeString>;
```

Comment:
What is the `len` here? The length of the entire transaction?

In general there are docs missing on what these parameters are.

`Bytes` should also be replaced by `Vec<u8>`. No reason to use this type here.

Reply:
I think we also could just add some doc:

```
# Panicking

Panics if the `encoded_call` can not be decoded as runtime `Call`.
```
Something like this. 

The runtime api would also panic if you pass some data that is not decodable.

Reply:
`len` is the length of the call. I dropped it, no needed anymore.